src/main/java/org/osc/controller/nsc/api/NeutronSdnRedirectionApi.java

```diff
@@ -92,95 +119,57 @@ public Long getInspectionHookTag(NetworkElement inspectedPort, InspectionPortEle
 
     @Override
     public void setInspectionHookTag(NetworkElement inspectedPort, InspectionPortElement inspectionPort, Long tag)
-            throws NetworkPortNotFoundException, Exception {
-        try (NeutronSecurityControllerApi neutronApi = new NeutronSecurityControllerApi(new Endpoint(this.vc))) {
-            InspectionHook inspectionHook = neutronApi.getInspectionHookByPorts(this.region, inspectedPort.getElementId(), inspectionPort);
-            inspectionHook.setTag(tag);
-            neutronApi.updateInspectionHook(this.region, inspectionHook);
-        }
+            throws Exception {
+
```

Comment:
implementation missing, also please see other methods that previously had an implementation